katacomb/katacomb/tests/test_uv_export.py

```diff
@@ -206,8 +203,8 @@ def sort_fn(x): return (cp[x].ant1_ix, cp[x].ant2_ix, cp[x].cid)
                 def _strip_strings(aips_keywords):
                     """ AIPS string are padded, strip them """
                     return {k: v.strip()
-                            if isinstance(v, six.string_types) else v
-                            for k, v in aips_keywords.iteritems()}
+                            if isinstance(v, str) else v
```

Comment:
I don't know the context here - is there a change that `v` might be a byte string?

Reply:
Much of my work converting Obit to python 3 was dealing with bytes/unicode string issues, and though I'm pretty sure I got them all properly converted to unicode- it probably pays to be safe- so I changed to:
`isinstance(v, (str, bytes))`